sources/packages/web/src/router/index.ts

```diff
@@ -40,6 +40,8 @@ const router = createRouter({
 // When tried to return true or route location instead of next with current version of vue-router
 // results in router not working as expected on page refresh.
 router.beforeResolve(async (to, _from, next) => {
+  // For now, all the page title is same, we can customize in future, if we want.
+  document.title = "StudentAid BC";
```

Comment:
I was wondering if that should come from any constant and be used in BCLogo.vue too for example. Thoughts?

Reply:
mm, actually I was under the assumption that this is for now and we may be customizing each router title in the future as usual.

@hellolynn-tbtb can you confirm the future plan?

Reply:
I don't have hard feeling to use a constant. I'm ok either way.

Reply:
If there is no intention now to be specific per portal why we cant just set the title in the index.html?
`<title>StudentAid BC</title>`

Reply:
Although I like it - likely low priority anyway for a specified page title. Let's go with something simple and what we know for now (e.g. StudentAid BC). My two cents. Thanks!

Reply:
ok... for now going with `title` tag